net.labymod.LabyMod.yml

```diff
@@ -0,0 +1,36 @@
+id: net.labymod.LabyMod
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: labymodlauncher
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --env=ELECTRON_DISABLE_SANDBOX='1'
```

Comment:
why would you do that?

Reply:
If I don't disable the Electron sandbox I get this error:
`The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /app/bin/chrome-sandbox is owned by root and has mode 4755.`
I can set mode 4755, but not root as owner. 
Since the whole project is already running in a sandbox, I didn't think it was too bad not to use Electron's sandbox.

Reply:
There is zypak for that. It's in the base app.

Reply:
Thanks, should be done